components/director/internal/domain/labeldef/resolver.go

```diff
@@ -49,6 +47,12 @@ func (r *Resolver) CreateLabelDefinition(ctx context.Context, in graphql.LabelDe
 		return nil, err
 	}
 
+	// TODO: Use LabelDefinitionInput
```

Comment:
Hmm, what does this TODO mean? the input is used, right?

Reply:
And also why did you move conversion to the top? If there will be transaction error, then there is no need to convert input to actual LabelDefinition.